Jepsen run on master ends up with 'Repository content has invalid line'

https://jenkins.scylladb.com/view/master/job/scylla-master/job/jepsen/job/jepsen-test-all/85/execution/node/110/log/

```
13:11:56  Traceback (most recent call last):
13:11:56  File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 3424, in node_setup
13:11:56  cl_inst.node_setup(_node, **setup_kwargs)
13:11:56  File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 4528, in node_setup
13:11:56  node.download_scylla_repo(scylla_repo_loader)
13:11:56  File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/decorators.py", line 64, in inner
13:11:56  return func(*args, **kwargs)
13:11:56  File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 1727, in download_scylla_repo
13:11:56  verify_scylla_repo_file(result.stdout, is_rhel_like=False)
13:11:56  File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/common.py", line 168, in verify_scylla_repo_file
13:11:56  assert valid_prefix, 'Repository content has invalid line: {}'.format(line)
13:11:56  AssertionError: Repository content has invalid line: [scylladb-4.6]
```

--- a/test-cases/jepsen/jepsen.yaml
+++ b/test-cases/jepsen/jepsen.yaml
@@ -15,6 +15,7 @@ nemesis_class_name: 'NoOpMonkey'
 scylla_linux_distro: 'debian-buster'
 user_prefix: 'jepsen'
 use_legacy_cluster_init: true
+scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/deb/debian/scylla-4.6-buster.list'
 jepsen_scylla_repo: 'https://github.com/scylladb/jepsen.git'
 jepsen_test_count: 5
 jepsen_test_run_policy: any

